front/plugins/csv_backup/script.py

```
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+# !/usr/bin/env python
```

Ruff (0.14.5)

The file is executable but no shebang is present

(EXE002)